xml/admin_saltcluster.xml

```diff
@@ -1141,14 +1142,28 @@ OSD_ID  HOST         STATE                    PG_COUNT  REPLACE  FORCE  STARTED_
    </step>
    <step>
     <para>
-     If the &adm; is a &smaster; at the same time, stop
-     <systemitem class="daemon">salt-master</systemitem>
-     <systemitem class="daemon">salt-minion</systemitem> services on the old
-     &adm;:
+     If the old &smaster; is also the only administration node in the cluster,
+     then manually move <filename>/etc/ceph/ceph.client.admin.keyring</filename>
+     and <filename>/etc/ceph/ceph.conf</filename> to the new &smaster;.
     </para>
+    </step>
+    <step>
+      <para>
+        Stop and disable the &smaster; &systemd; service on the old &smaster; node:
+      </para>
 <screen>
 &prompt.smaster;systemctl stop salt-master.service
+&prompt.smaster;systemctl disable salt-master.service
+</screen>
+   </step>
+   <step>
+     <para>
+       If the old &smaster; node is no longer in the cluster, also stop and
+       disable the &sminion; &systemd; service:
+     </para>
+<screen>
 &prompt.smaster;systemctl stop salt-minion.service
+&prompt.smaster;systemctl disable salt-minion.service
```

Comment:
WARNING: Don't stop or disable the salt-minion.service if the old `&smaster;` node has any Ceph daemons (MON, MGR, OSD, MDS, gateway, monitoring) running on it.

Reply:
You want this warning added?

Reply:
Yes. Right there where I wrote it.